docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+By default, CloudTrail logs API calls that were made in the last 90 days, but not log requests made to objects. You can see bucket-level events on the CloudTrail console. However, you can't view data events (Amazon S3 object-level calls) there—you must parse or query CloudTrail logs for them. 
+
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail logs for the eventnames `GetPublicAccessBlock` and `DeletePublicAccessBlock`
+
+### CloudTrail: Public S3 Object
+You can also get CloudTrail logs for object-level Amazon S3 actions. To do this, enable data events for your S3 bucket or all buckets in your account. When an object-level action occurs in your account, CloudTrail evaluates your trail settings. If the event matches the object that you specified in a trail, the event is logged. 
+
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail logs for the eventnames `GetObjectAcl` and `PutObjectAcl`
+
+### VPC Flow Logs
+VPC Flow Logs is a feature that enables you to capture information about the IP traffic going to and from network interfaces in your VPC. This can be useful for IP addresses discovered within CloudTrail to determine the types of external connections to any public resources. 
+
+For further information and steps, including querying with Athena, please refer to the [AWS Documentation for VPC Flow Logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-athena.html). It is recommended that Athena analysis be included in a separate playbook and linked to other relevant items. 
+
+### Endpoint / Host Based
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail for `PutObject` and  `DeleteObject` requests from public IP addresses
+
+- Review EC2 operating system and application logs for inappropriate logins, installation of unknown software, or the presence of unrecognized files. 
+
+- It is highly recommended to have a third-party host-based intrusion detection system (HIDS) solution (such as OSSEC, Tripwire, Wazuh, [Amazon Inspector](https://aws.amazon.com/inspector/), other) 
+
+## Containment
+
+### S3 Block Public Access
+aws s3api put-public-access-block --bucket bucket-name-here --public-access-block-configuration "BlockPublicAcls=true,IgnorePublicAcls=true,BlockPublicPolicy=true,RestrictPublicBuckets=true"
+
+You can also review [Blocking public access to your Amazon S3 storage](https://aws.amazon.com/s3/features/block-public-access/) for additional details on blocking public S3 access across your account.
+
+## Eradication
+
```

Comment:
Did you want to include processes/documents for EC2 and CodeCommit?

Reply:
Added documentation for EC2 and CodeCommit.